experimental/devices/mfrc522/mfrc522.go

```diff
@@ -514,156 +684,19 @@ func (r *Dev) SelectTag(serial []byte) (byte, error) {
 	return blocks, nil
 }
 
-// StopCrypto stops the crypto chip.
-func (r *Dev) StopCrypto() error {
+// stopCrypto stops the crypto chip.
+func (r *Dev) stopCrypto() error {
```

Comment:
I would move this method (and alike) to the `lowlevel` package rather than making them private.

Reply:
That's a fair comment and I'm fine with this idea. I used something similar with lepton, moving all commands to a subpackage cci https://periph.io/x/periph/devices/lepton/cci.

Here I feel this would supersede commands .